### DISCUSSION TITLE
PureVPN & port forwarding

### DISCUSSION THREAD

**Discussion by madcowGit** [haugene/docker-transmission-openvpn#2539] [Ideas]
It took me quite a while to figure this out, so I'm placing it here for anyone who is looking for it.

I have PureVPN and wanted to configure port forwarding through the VPN (which requires the addon from PureVPN).
However, the ports within the docker container would not open.  

The issue for me was that I had specified a PUID and PGID in my docker compose. After removing those options my ports opened up as expected.
http://haugene.github.io/docker-transmission-openvpn/config-options/#user_configuration_options

I believe it has something to do with how the iptables are modified during initialization. But I havent looked into it any further.

FYI:
- I actually use https://github.com/jsloan117/docker-openvpn-client, which is based on this image.
- I had identical behaviour with other images (e.g. gluetun)

**Comment by LioLynf** [comment]
Hi ! Thanks a lot for your post.
Got the same problem with Hotspot Shield (so custom provider in my composer)
WebUI online but unable to connect to tracker.
Remove PUID and PGID resolved the problem !

**Comment by PradipCodes** [comment]
You need to use the servers where port forwarding are enabled. You have to download ovpn files from pure vpn manually and make the openvpn use that server.
